EIPS/eip-template.md

```diff
@@ -0,0 +1,235 @@
+
+```solidity
+event RevocationStatusesChanged(
+    address[] indexed namespaces,
+    bytes32[] indexed lists,
+    bytes32[] indexed keys,
+    bool[] revoked
+);
+```
+
+**ListOwnerChanged**
+**MUST** be emitted when `changeListOwner` or `changeListOwnerSigned` was successfully executed.
+
+```solidity
+event ListOwnerChanged(
+    address indexed namespace,
+    address indexed newOwner,
+    bytes32 indexed list
+);
+```
+
+**DelegateAdded**
+**MUST** be emitted when `addListDelegate` or `addListDelegateSigned` was successfully executed.
+
+```solidity
+event ListDelegateAdded(
+    address indexed namespace,
+    address indexed delegate,
+    bytes32 indexed list
+);
+```
+
+**DelegateRemoved**
+**MUST** be emitted when `removeListDelegate` or `removeListDelegateSigned` was successfully executed.
+
+```solidity
+event ListDelegateRemoved(
+    address indexed namespace,
+    address indexed delegate,
+    bytes32 indexed list
+);
+```
+
+### Meta transactions
+
+This section uses the following terms: 
+
+**`transaction signer`:** An Ethereum address that signs arbitrary data for the contract to execute **BUT** does not commit the transaction. 
+**`transaction sender`**: An Ethereum address that takes signed data from a **transaction signer** and commits it wrapped with its own signature to the smart contract.
+
+An address (**transaction signer**) **MAY** be able to deliver a signed payload off-band to another address (**transaction sender**) that initiates the Ethereum interaction with the smart contract. The signed payload **MUST** be limited to be used only once ([Signed Hash](#Signed-Hash) + [nonces](#Nonce)).
+
+#### Signed Hash
+
+The signature of the **transaction signer** **MUST** conform [EIP-712](/EIPS/eip-712), in its state that is proposed in [`ethereum/EIPs/issues/5475`](https://github.com/ethereum/EIPs/issues/5475). This helps users understand what the payload they're signing consists of & it improves the protection against replay attacks.
+
+#### Nonce
+
+This EIP **RECOMMENDS** the use of a **dedicated nonce mapping** for meta transactions. If the signature of the **transaction sender** and its meta contents are verified, the contract increases a nonce for this **transaction signer**. This effectively removes the possibility for any other sender to execute the same transaction again with another wallet. 
+
+## Rationale
+
+### Why the concept of namespaces?
+> This provides every Ethereum address a reserved space, without the need to actively claim it in the contract. Initially addresses only have owner access in their own namespace.
+
+### Why does a namespace always represent the initial owner address? 
+> The change of an owner of a list shouldn't break the link to a revocation key in it, as already existing off-chain data may depend on it. 
+
+## Backwards Compatibility
+Not applicable
+
+## Reference Implementation
+tba
```

Comment:
```suggestion
```
Required: The Reference Implementation section is optional and should be left empty when there is no content.